packages/sdk/src/evm/common/legacy.ts

```diff
@@ -7,12 +7,10 @@ export async function getPrebuiltInfo(
 ): Promise<{ type: string; version: number } | undefined> {
   try {
     const contract = new Contract(address, IThirdwebContractABI, provider);
-    const [type, version] = await Promise.all([
-      utils
-        .toUtf8String(await contract.contractType()) // eslint-disable-next-line no-control-regex
-        .replace(/\x00/g, ""),
-      await contract.contractVersion(),
-    ]);
+    const version = await contract.contractType();
+    const type = utils
+      .toUtf8String(version) // eslint-disable-next-line no-control-regex
+      .replace(/\x00/g, "");
```

Comment:
this looks potentially wrong - version used to be `contractVersion()` and now became `contractType()`?

Reply:
oops I'll fix it

Reply:
https://github.com/thirdweb-dev/js/pull/1648/commits/6d0588efeee56f83b1845b157e712f2a4baeb2ab